kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -272,8 +271,7 @@ class ResidualBasedBlockBuilderAndSolver
             }
         }
 
-        const double stop_build = timer.ElapsedSeconds();
-        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", (this->GetEchoLevel() >= 1 && rModelPart.GetCommunicator().MyPID() == 0)) << "Build time: " << stop_build - start_build << std::endl;
+        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", (this->GetEchoLevel() >= 1 && rModelPart.GetCommunicator().MyPID() == 0)) << "Build time: " << timer.ElapsedSeconds() << std::endl;

```

Comment:
```suggestion
        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", this->GetEchoLevel() >= 1) << "Build time: " << timer.ElapsedSeconds() << std::endl;

```

Reply:
guys, why are u taking out the mpi related stuff??

Reply:
This is a leftover, the logger handles this